src/main/java/org/itsallcode/openfasttrace/ImportSettings.java

```diff
@@ -82,13 +95,43 @@ public static Builder builder()
      */
     public static class Builder
     {
+        private final List<Path> inputs = new ArrayList<>();
         private FilterSettings filter = FilterSettings.createAllowingEverything();
         private List<PathConfig> pathConfigs = new ArrayList<>();
 
         private Builder()
         {
         }
 
+        /**
+         * Add a list of input paths to be scanned for importable artifacts
+         * 
+         * @param inputs
+         *            input paths to be scanned
+         * @return <code>this</code> for fluent programming
+         */
+        public Builder addInputs(final List<Path> inputs)
+        {
+            this.inputs.addAll(inputs);
+            return this;
+        }
+
+        /**
+         * Add one or more input paths to be scanned for importable artifacts
+         * 
+         * @param inputs
+         *            input paths to be scanned
+         * @return <code>this</code> for fluent programming
+         */
+        public Builder addInputs(final Path... inputs)
+        {
+            for (final Path input : inputs)
```

Comment:
You could do: `this.addInputs(asList(inputs));`

Reply:
That would add another Import and creates an intermediate `ArrayList` so I consider this solution better here.